tests/integration/cases/apache_maven_local_paths_in_configuration_without_dep_resolution/config.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 - 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Looks to me like this file hasn't changed at all. Is it that fun little problem with copyright headers again?

Reply:
That's true. The year shouldn't change. 
> Is it that fun little problem with copyright headers again?
Maybe :sweat_smile:. I honestly don't remember how it ended up like that.